language/ruby/language_test.go

```diff
@@ -62,3 +64,75 @@ func TestLanguageImportPath(t *testing.T) {
 		ExpectedImportPath: "../lib/plain",
 	})
 }
+
+func TestParseSymflowerTestOutput(t *testing.T) {
+	type testCase struct {
+		Name string
+
+		Data string
+
+		ExpectedTestsTotal int
+		ExpectedTestsPass  int
+	}
+
+	validate := func(t *testing.T, tc *testCase) {
+		t.Run(tc.Name, func(t *testing.T) {
+			actualTestsTotal, actualTestsPass, actualErr := parseSymflowerTestOutput(bytesutil.StringTrimIndentations(tc.Data))
+			require.NoError(t, actualErr)
+
+			assert.Equal(t, tc.ExpectedTestsTotal, actualTestsTotal)
+			assert.Equal(t, tc.ExpectedTestsPass, actualTestsPass)
+		})
+	}
+
+	validate(t, &testCase{
+		Name: "Tests with failures",
+
+		Data: `
+			Bundle complete! 3 Gemfile dependencies, 7 gems now installed.
+			Bundled gems are installed into ` + "`" + `./vendor/bundle` + "`" + `
+			Run options: --seed 64090
+
+			# Running:
+
+			F.E...
+
+			Finished in 0.001245s, 4820.4968 runs/s, 4017.0806 assertions/s.
+
+  			1) Failure:
+			TestBalancedBrackets#test_has_balanced_brackets6 [test/balanced_brackets_test.rb:38]:
+			Expected: true
+  			Actual: false
+
+  			2) Error:
+			TestBalancedBrackets#test_has_balanced_brackets5:
+			NoMethodError: undefined method ` + "`" + `has_balancedbrackets' for an instance of TestBalancedBrackets
+    			test/balanced_brackets_test.rb:31:in ` + "`" + `test_has_balanced_brackets5'
+
+			6 runs, 5 assertions, 1 failures, 1 errors, 0 skips
+			rake aborted!
+			Command failed with status (1)
+		`,
+		ExpectedTestsTotal: 6,
+		ExpectedTestsPass:  4,
```

Comment:
Why are there 5 assertions if two of the tests failed? I guess because failed assertions also count?